pkg/sdn/plugin/pod_linux.go

```diff
@@ -360,6 +362,34 @@ func (m *podManager) ipamGarbageCollection() {
 	}
 }
 
+// TODO copied & modified from upstream. Remove once upstream func is exported.
+func constructPodPortMapping(internalPod *api.Pod, podIP net.IP) (*hostport.PodPortMapping, error) {
```

Comment:
Just put it in here. I will move that to our kube rebase branch on the next rebase.